setup.py

```diff
@@ -57,6 +57,7 @@ def version(path):
     "fsspec",
     "pandas",
```

Comment:
I think we can get rid of `pandas` dependency, right?

Reply:
i still use your older version of code which uses get_handle for different remote filepaths. so only for that i do need pandas right now. if we can refactor that piece too somehow then ya we should be able to get rid of this piece.